polkadot/zombienet_tests/functional/0016-verify-included-events.js

```diff
@@ -0,0 +1,42 @@
+function parse_pjs_int(input) {
+    return parseInt(input.replace(/,/g, ''));
+}
+
+async function run(nodeName, networkInfo) {
+    const { wsUri, userDefinedTypes } = networkInfo.nodesByName[nodeName];
+    const api = await zombie.connect(wsUri, userDefinedTypes);
+
+    let blocks_per_para = {};
+
+    await new Promise(async (resolve, _) => {
+        let block_count = 0;
+        const unsubscribe = await api.query.system.events((events) => {
+            block_count++;
+
+            events.forEach((record) => {
+                const event = record.event;
+
+                if (event.method != 'CandidateIncluded') {
+                    return;
+                }
+
+                let included_para_id = parse_pjs_int(event.toHuman().data[0].descriptor.paraId);
+                if (blocks_per_para[included_para_id] == undefined) {
+                    blocks_per_para[included_para_id] = 1;
+                } else {
+                    blocks_per_para[included_para_id]++;
+                }
+            });
+
+            if (block_count == 12) {
+                unsubscribe();
+                return resolve();
+            }
+        });
+    });
+
+    console.log(`Result: 2000: ${blocks_per_para[2000]}, 2001: ${blocks_per_para[2001]}`);
+    return (blocks_per_para[2000] == 6 && blocks_per_para[2001] == 1);
```

Comment:
Because I was testing some stuff and carelessly committed messed up version of the test :-1: 

I've fixed the whole test. I'm still worried it will be flaky but let's see.